.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,17 @@
-<!-- 🎉 Thank you for submitting a PR! Before submitting, please review the following checklist: -->
+<!-- 0. 🎉 Thank you for submitting a PR! -->
 
-- [ ] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](/docs/CONTRIBUTING.md)
-- [ ] ✅ I've added tests where applicable to test my new functionality.
-- [ ] 📖 I've made sure that my contracts are well-documented.
-- [ ] 🎨 I've run the JS/Solidity linters (`npm run lint:all:fix`) and fixed any issues.
-
-<!-- **Does this close any open issues?** If so, list them here. -->
+<!-- 1. **Does this close any open issues?** If so, list them here. If not, remove the `Fixes #` line. -->
 
 Fixes #
 
----
-
 # 🚀 Description
 
-<!-- Describe the changes introduced in this pull request -->
-<!-- Include any context necessary for understanding the PR's purpose. -->
\ No newline at end of file
+<!-- 2. Describe the changes introduced in this pull request -->
+<!--    Include any context necessary for understanding the PR's purpose. -->
+
+<!-- 3. Before submitting, please review the following checklist: -->
+
+- [ ] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](/docs/CONTRIBUTING.md)
+- [ ] ✅ I've added tests where applicable to test my new functionality.
+- [ ] 📖 I've made sure that my contracts are well-documented.
+- [ ] 🎨 I've run the JS/Solidity linters (`npm run lint:all:fix`) and fixed any issues.
```

Comment:
While we're at it, I'd move the command in parentheses to the end of the sentence, after "fixed any issues".